src/com/fundation/search/controller/ControlCommand.java

```diff
@@ -71,44 +74,43 @@ public ControlCommand(String[] inputCommands) {
         this.searchCommand = new SearchCommand(inputCommands);
 
         // Initialize CommandCriteria
-        this.commandCriteria = new CommandCriteria(searchCommand.getMapToSearch());
-
-        // Initialize Validator
-        this.validateInputs = new Validator();
-
-        // Initialize Converter
-        this.converter = new Convertor();
-
-        this.listenCommands();
-
+        if (!searchCommand.getMapToSearch().isEmpty()) {
+            this.commandCriteria = new CommandCriteria(searchCommand.getMapToSearch());
+            this.converter = new Convertor();
+            this.commandView = new CommandView();
+            this.validator = new Validator();
+            if (validateAllInputs()) {
+                this.listenCommands();
+            }
+        }
         LOOGER.info("Constructor exit");
     }
 
     /***
-     * This method check the event button "search" and fill Data from User Interface.
-     * then the inputs are insert for validate.
+     * This method  fill Data from Command Line in Search Criteria object.
      */
     private void listenCommands() {
         LOOGER.info("Action Search Button entry");
 
-        Date[] dateCreation = converter.convertDate(dateValidation(commandCriteria.getDateCreation()));
-        Date[] dateModified = converter.convertDate(dateValidation(commandCriteria.getDateModified()));
-        Date[] dateLastAccess = converter.convertDate(dateValidation(commandCriteria.getDateLastAccess()));
-        String[] getSize = splitGetSize(commandCriteria.getSize());
+        FileTime[] dateCreation = converter.convertStringToListFileTime(commandCriteria.getDateCreation());
+        FileTime[] dateModified = converter.convertStringToListFileTime(commandCriteria.getDateModified());
+        FileTime[] dateLastAccess = converter.convertStringToListFileTime(commandCriteria.getDateLastAccess());
+        String[] getSize = converter.splitGetSize(commandCriteria.getSize());
+
         // Adding to SearchCriteria and Validating some data
         this.searchCriteria = new com.fundation.search.controller.builder.SearchCriteriaBuilder()

```

Comment:
You need declare this library in the header "com.fundation.search.controller.builder"